cmd/zpool/Makefile.am

```diff
@@ -129,10 +129,56 @@ zpoolconfdefaults = \
 	test_progress \
 	test_ended
 
+zpoolcompatdir = $(pkgdatadir)/compatibility.d
+
+dist_zpoolcompat_DATA = \
+	compatibility.d/compat-2018 \
+	compatibility.d/compat-2019 \
+	compatibility.d/compat-2020 \
+	compatibility.d/compat-2021 \
+	compatibility.d/freebsd-11.0 \
+	compatibility.d/freebsd-11.2 \
+	compatibility.d/freebsd-11.3 \
+	compatibility.d/freenas-9.10.2 \
+	compatibility.d/grub2 \
+	compatibility.d/o3x-1.7.0 \
+	compatibility.d/o3x-1.8.1 \
+	compatibility.d/o3x-1.9.3 \
+	compatibility.d/openzfs-2.0-freebsd \
+	compatibility.d/openzfs-2.0-linux \
+	compatibility.d/zol-0.6.5 \
+	compatibility.d/zol-0.7 \
+	compatibility.d/zol-0.8
+
+# canonical <- alias symbolic link pairs
+# eg: "compat-2018" is a link to "zol-0.6.5"
```

Comment:
```suggestion
# eg: "2018" is a link to "compat-2018"
```
Since the compat-YYYY files are all canonical now, the comment didn't make much sense.

Reply:
ACK, thanks.